spec/plugins/logging_spec.lua

```diff
@@ -149,7 +151,7 @@ describe("Logging Plugins", function()
     -- Making sure it's alright
     assert.are.same("127.0.0.1", log_message.client_ip)
   end)
-
+  
```

Comment:
Some projects have *very* strict policies against white-spaces and sometimes I think we should too. Just configure your editor or run a pre-commit hook.